src/Aspire.Hosting.Sdk/SDK/Sdk.targets

```diff
@@ -37,6 +49,6 @@
   also contains Aspire.Hosting.Orchestration and vice versa. Once Aspire/DCP are public
   we can move it to WorkloadManifest.targets to ensure future correctness.
   -->
-  <Import Project="Sdk.targets" Sdk="Aspire.Hosting.Orchestration" />
+  <Import Project="Sdk.targets" Sdk="Aspire.Hosting.Orchestration" Condition="'$(_SuppressImportAspireHostingOrchestrationTargets)' != 'true'" />
```

Comment:
I don't love this change, as we are now leaking an implementation detail of how we build and test stuff in the repo into our shipped SDK. That said, I'd have to play a bit more with this to see if there are other alternatives. Why wasn't this needed before?

Reply:
We never imported the Sdk.targets before. But with the transitively change mentioned above, we now need to import the Sdk.targets file from our AppHost projects in the repo (and the new test I added). Importing the `Aspire.Hosting.Orchestration` targets doesn't work here, which is why I added the opt out mechanism.

This is roughly equivalent to what we have in `SkipAspireWorkloadManifest`:

https://github.com/dotnet/aspire/blob/0d1f8ea7750b69c8f221816077e9e5a5baa1bc0a/src/Microsoft.NET.Sdk.Aspire/WorkloadManifest.in.targets#L3-L6

Reply:
fair enough.